style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  background-color: mistyrose;
+  margin-left: 150px;
+  margin-right: 200px;
+}
+
+h1 {
+  font-family: cursive;
+  font: 700px;
```

Comment:
- The font has to be qualified
- Either font-family, font-weight, font-size etc...
- Saying font: 700px; doesn't style anything, please try to change that